arch/arm/core/aarch32/cortex_a_r/tcm.c

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (c) 2019, Antmicro
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include <zephyr.h>
+#include <cmsis_cp15.h>
+
+void z_arm_tcm_disable_ecc(void)
+{
+	uint32_t actlr;
+
+	actlr = __get_ACTLR();
+	actlr &= ~0xe000000;
```

Comment:
Given that these bits are defined in the CMSIS, please make use of the definitions:
https://github.com/zephyrproject-rtos/cmsis/blob/a2b82e79bd8ac156fb283d5f52fa0a2373962cf9/CMSIS/Core_R/Include/core_cr.h#L511-L518

```suggestion
	actlr &= ~(ACTLR_ATCMPCEN_Msk | ACTLR_B0TCMPCEN_Msk | ACTLR_B1TCMPCEN_Msk);
```

Reply:
done